mkdocs.yml

```diff
@@ -1,204 +1,237 @@
 site_name: EdgeX Foundry Documentation
-docs_dir: ./docs_src
-site_dir: ./docs/3.1
 site_description: 'Documentation for use of EdgeX Foundry'
-site_author: 'Michael Johanson'
-site_url: 'https://docs.edgexfoundry.org/3.1'
+site_author: 'Jim White'
+site_url: 'https://github.com/edgexfoundry/edgex-docs'
```

Comment:
Why change this to the repo URL? Should be `https://docs.edgexfoundry.org/`

Reply:
@lenny-intel This also has to do with experimentation of the new versioning mechanism.  Mike plugin is looking for the versions based on the site_url at first.  This may not be the way this ends up.  Again, @ernestojeda is helping me with this part.  Once we have it working, I will put it back if it is not required.

Reply:
@jpwhitemn  I did some testing locally with the built site (`make build`) and using the previous value of `https://docs.edgexfoundry.org/3.1` worked fine for me.